openupgrade_scripts/scripts/hr_recruitment/16.0.1.1/post-migration.py

```diff
@@ -0,0 +1,13 @@
+from openupgradelib import openupgrade
+
+translations_to_delete = [
+    "group_hr_recruitment_user",
```

Comment:
The translations you have to delete are the mail template ones:

```suggestion
    "email_template_data_applicant_congratulations",
    "email_template_data_applicant_interest", 
```